.gitignore

```diff
@@ -6,6 +6,11 @@
 .flattened-pom.xml
 */dependency-reduced-pom.xml
 
+# eclipse stuff
```

Comment:
Is the eclipse stuff not exclude inside the global ignore file?

Reply:
I've reverted changes in .gitignore file